tools/ktx/command_create.cpp

```diff
@@ -590,13 +590,25 @@ Create a KTX2 file from various input files.
     The number of input-files specified must match the expected number of input images
     based on the used options.
 
-    The following options are available:
+@section ktx\_create\_options OPTIONS
+  @subsection ktx\_create\_options\_general General Options
+    The following are available:
     <dl>
         <dt>\--format &lt;enum&gt;</dt>
-        <dd>KTX format enum that specifies the image data format.
+        <dd>KTX format enum that specifies the data format of the images in the created texture.
             The enum names are matching the VkFormats without the VK_FORMAT_ prefix.
-            The VK_FORMAT_ prefix is ignored if present.<br />
-            When used with --encode it specifies the target format before the encoding step.
+            The VK_FORMAT_ prefix is ignored if present. Case insensitive. Required.<br />
+            <br />
+            If the format is an ASTC format a texture object with the target format
+            @c R8G8B8_{SRGB,UNORM} or  @c R8G8B8A8_{SRGB,UNORM} is created
+            then encoded to the specified ASTC format. The latter format is chosen if alpha
+            is present in the input. @c SRGB or @c UNORM is chosen depending on the
+            specified ASTC format. The ASTC-specific and common encoder options listed
+            @ref ktx_create_options_encoding "below" become valid, otherwise they are ignored.
+            <!--This matches the functionality of the @ref ktx_encode "ktx encode" command
+            when an ASTC format is specified.<br /> -->
+            <br />
+            When used with @b \--encode it specifies the target format before the encoding step.
```

Comment:
Does this now mean --encode and ASTC format are valid options now?

Reply:
ok I got this one now, This paragraph doesn't below to the ASTC one.

Reply:
Correct. I put the ASTC paragraph first in an attempt to avoid the confusion you had. I found it far more confusing when the ASTC paragraph came after mention of `--encode`.